pyupgrade.py

```diff
@@ -1287,27 +1371,25 @@ class Block(
 ):
     __slots__ = ()
 
-    def _initial_indent(self, tokens):
+    def _initial_indent(self, tokens):  # type: (List[Token]) -> int
         if tokens[self.start].src.isspace():
             return len(tokens[self.start].src)
         else:
             return 0
 
-    def _minimum_indent(self, tokens):
-        block_indent = None
+    def _minimum_indent(self, tokens):  # type: (List[Token]) -> int
+        block_indent = sys.maxsize
```

Comment:
oof this is going to do some nasty things if this function is broken -- I'd rather keep this `None` and if you need to convince mypy then `assert block_indent is not None` at the end of the function

Reply:
sure